lib/binary_to_decimal.rb

```diff
@@ -6,4 +6,9 @@
 # the algorithm you devised in class.
 def binary_to_decimal(binary_array)
   raise NotImplementedError
+  decimal = 0
+  8.times do |i|
```

Comment:
Good job with the do ... loop. Can you think of a way to generalize the algorithm so that it works for n bits?